examples/appdir/tailwind.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     "./src/pages/**/*.{js,ts,jsx,tsx}",
     "./src/components/**/*.{js,ts,jsx,tsx}",
     "./src/app/**/*.{js,ts,jsx,tsx}",
+    "./node_modules/@uploadthing/react/src/components/**",
```

Comment:
might look a bit hacky but I do genuinely think this is the most optimal way to include tailwind classes from a lib into an app that uses tailwind

Reply:
Maybe instead of requiring users to manually add this line, it would be possible to create a function that would automatically append that line by simply accepting config as argument, mutating it inside and then returning back

Something like

```typescript
import { withUt } from 'uploadthing/tailwind'

export default withUt({
  // whatever TW config
})
```

The same with CJS

```javascript
const { withUt } = require('uploadthing/tailwind')

module.exports = withUt({
  // whatever TW config
})
```